bindings/wasm/lib/import-model.ts

```diff
@@ -137,28 +156,130 @@ export async function importManifold(uri: string): Promise<Manifold> {
  *
  * Other errors will be re-thrown for the caller to handle.
  *
+ * @group Conversion Functions
  * @returns A valid Manifold object.
  */
 export function gltfNodeToManifold(
-    document: GLTFTransform.Document, node: GLTFTransform.Node): Manifold {
-  const converted: Array<Manifold> = [];
-  node.traverse(child => {
-    const manifold = gltfMeshToManifold(document, child);
-    if (manifold) converted.push(manifold);
-  });
-  if (!converted.length) {
-    throw new Error(`Model contains no manifold geometry.`);
+    document: GLTFTransform.Document, node?: GLTFTransform.Node): Manifold {
```

Comment:
I don't think this needs to be public either - `importManifold` is a nice high-level function; if that's not adequate, I think they should probably go all the way down to dealing with each mesh individually.